packages/yourloops/pages/login/login.tsx

```diff
@@ -134,15 +163,22 @@ function Login(props: RouteComponentProps): JSX.Element {
 
   return (
```

Comment:
Commented functions above can be removed ?

Reply:
i am going to change them soon in another PR